src/pages/CropSidebar/CropSidebar.js

```diff
@@ -43,8 +43,9 @@ const CropSidebar = ({
   isListView,
   from,
   setGrowthWindow,
-  toggleComparisonView,
-  toggleListView,
+  setComparisonView,
+  setIsListView,
```

Comment:
do you need both of these? seems like unneeded logic since there's only two options

Reply:
in one if block on line 314 it's setting isListView and in the next it uses isComparisonView but it should only use one value. this will reduce the number of props by 2. i'd say either make a new variable called view and have it be a string 'list' or 'comparison' or have it be a boolean and know true is one and false is the other. having 4 props in CropSidebar for whether it's list or comparison view seems excessive

Reply:
i also don't think `from === 'myCoverCropListStatic'` is ever true so that whole if block on line 314 can probably be removed but double check first